lib/util/git/index.ts

```diff
@@ -402,6 +402,23 @@ export async function syncGit(): Promise<void> {
     logger.warn({ err }, 'Cannot retrieve latest commit');
   }
   config.currentBranch = config.currentBranch || (await getDefaultBranch(git));
+  // istanbul ignore if
```

Comment:
I don't have time to figure out how to test this and need a test for forking-renovate ASAP

Reply:
Ok, but we should test this in near future.

Maybe @zharinov  or @JamieMagee  can help.